design-library/source/_patterns/01-atoms/04-blog-tag/blog-tag.json

```diff
@@ -1,3 +1,7 @@
 {
-  "tag_name": "CATEGORY 1"
-}
\ No newline at end of file
+  "tag_name": "Category 1",
+  "tag_icon_url": "/images/svg/drupal-2.svg",
+  "tag_color": "#0a509b",
```

Comment:
It would be a good idea to define the various colors somewhere in SCSS and try to specify the color as a class name or something here. We could add different JSON variations for the different colors, e.g. "blog-tag~1.json" and "blog-tag~2.json"